lib/galaxy/model/store/ro_crate_utils.py

```diff
@@ -220,7 +220,77 @@ def _add_workflows(self, crate: ROCrate):
 
             crate.license = self.workflow.license or ""
             crate.mainEntity["name"] = self.workflow.name
-            crate.mainEntity["subjectOf"] = cwl_wf
+
+            # Adding the creator information
+            if hasattr(self.workflow, 'creator_metadata') and self.workflow.creator_metadata:
+                creators = self.workflow.creator_metadata
+                if creators and isinstance(creators, list) and len(creators) > 0:
+                    first_creator = creators[0]
+                    creator_entity = crate.add(
+                        ContextEntity(
+                            crate,
+                            first_creator.get('identifier', ''),  # Default to empty string if identifier is missing
+                            properties={
+                                "@type": "Person",
+                                "name": first_creator.get('name', ''),  # Default to empty string if name is missing
+                                "orcid": first_creator.get('identifier', ''),  # Assuming identifier as orcid, or adjust accordingly
+                            },
+                        )
+                    )
+                    crate.mainEntity.append_to("creator", creator_entity)
+
+            # Add CWL workflow entity if exists
+            crate.mainEntity["subjectOf"] = cwl_wf if cwl_wf else ""
```

Comment:
The empty string is quite suboptimal, why was this not a problem before ? Can you just skip entries if we don't have them ?

Reply:
Yes I think if it's not there it can just be skipped